website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -55,6 +55,12 @@ We will use the `v1.3.0` release version in this example.
    Deploys](https://app.netlify.com/sites/operator-sdk/settings/deploys#branches).
    This will watch for this branch when there are changes on Github
    (creating the branch, or adding a commit).
+1. Kick off a new build of the [ansible-operator-base
+   image](https://quay.io/repository/operator-framework/ansible-operator-base)
+   by running the
+   [deploy-manual](https://github.com/operator-framework/operator-sdk/actions/workflows/deploy-manual.yml)
+   GitHub action.  After the image is built, check the security scan
+   results under `Child Manifests`.
```

Comment:
Not sure I'd make this a requirement for doing a release. These can be done at any point.

Reply:
It can be done at any time, but it seems like it should always be done right before a release. Can we make it a requirement until we come up with some other process that will guarantee it is done when it needs to be done?

Reply:
@varsha I'd like to get this into 1.13.0, do you think we are good to merge as is? @jmrodri I'll revert if I'm wrong

Reply:
I wouldn't mind having this as a pre-req for release for now. Its better to have it verified that we are on latest before a release though we can do it anytime too. Similar to how we update olm biodata (we have it as pre-req before a release, though we can have a PR to do to anytime a new OLM version is out).